server/portal/apps/workspace/api/views.py

```diff
@@ -441,7 +441,8 @@ def getPrivateApps(self, user):
 
     def getPublicApps(self, user):
         tapis = user.tapis_oauth.client
-        apps_listing = tapis.apps.getApps(select="version,id,notes", search="(enabled.eq.true)", listType="SHARED_PUBLIC")
+        # Updated to search only for the enabled version of any given app
+        apps_listing = tapis.apps.getApps(select="version,id,notes", search="(versionEnabled.eq.true)", listType="SHARED_PUBLIC")
```

Comment:
Great! Can you add this to `getPrivateApps` as well?

Reply:
Yep! Just added and pushed. (I had to make sure tests still passed.)